src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -38,6 +38,10 @@ message ResourceCommand {
     // Clients must return any value provided by the server when invoking
     // the command.
     optional google.protobuf.Value parameter = 4;
+    // A flag that indicates whether the command is highlighted in the UI.
+    bool is_highlighted = 5;
+    // Optional icon content. SVG path is displayed in the UI.
+    optional string icon_content = 6;
```

Comment:
An example of icon content here is:

```xml
<path d="M10.5 8.91a.5.5 0 0 0-1 .09v4.6a.5.5 0 0 0 1-.1V8.91Zm.3-2.16a.75.75 0 1 0-1.5 0 .75.75 0 0 0 1.5 0ZM18 10a8 8 0 1 0-16 0 8 8 0 0 0 16 0ZM3 10a7 7 0 1 1 14 0 7 7 0 0 1-14 0Z"/>
```

That is, it's an SVG path without any size or bounds information. There's an implicit connection between that string and the `IconSize` value used here.

The consuming code adds that dimension information via `IconSize.Size20`.

If we use SVG to allow icons to be defined in this way, we should make sure we at least document this expectation.